performance/models.py

```diff
@@ -7,18 +7,19 @@
 
 def eqprs_file_path(instance, filename):
     return (
-        f"eqprs_uploads/{uuid.uuid4()}/{filename}"
+        # f"eqprs_uploads/{uuid.uuid4()}/{filename}"
+        f"eqprs_uploads/{filename}"
```

Comment:
why are you removing the UUID? The point of the UUID is to ensure that the same filename doesn't replace an earlier upload's file

Reply:
perhaps we should document that

Reply:
I'm getting error with the UUID and I'll need your help.